gobyexample/server.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func main() {
+	addr := ":4242"
+	fmt.Printf("Open your web browser and visit http://127.0.0.1%s\n", addr)
+	err := http.ListenAndServe(addr, http.FileServer(http.Dir(".")))
```

Comment:
The usual pattern is `log.Fatal(http.ListenAndServe(...))`, since `ListenAndServe` only returns a non-nil error.